lib/ansiblelint/utils.py

```diff
@@ -863,12 +878,10 @@ def get_playbooks_and_roles(options=None):
             playbooks.append(normpath(p))
             continue
 
-        if options.verbosity:
-            print('Unknown file type: %s' % normpath(p))
+        _logger.info('Unknown file type: %s', normpath(p))
```

Comment:
It looks like you're changing the logic here. It's probably a warning

Reply:
I do not think so because the logging will happen or not based on verbosity level, is just that now we rely on logging to make the decision.